test/data/config/version=5.0.0.yml

```diff
@@ -126,7 +126,9 @@ tables_fail_threshold: "java.lang.Integer"
 trace_type_repair_ttl: "org.apache.cassandra.config.DurationSpec.IntSecondsBound"
 minimum_replication_factor_warn_threshold: "java.lang.Integer"
 page_size_warn_threshold: "java.lang.Integer"
-role_manager: "java.lang.String"
```

Comment:
ok but ... this is not true for 5.0.0, right? We just changed something for 5.0.0.yaml.

I think 5.0-alpha1 should be just renamed to 5.0.0.yaml and then 5.0.0.yaml copied to 5.1-alpha1 and change what is different for this patch.

Reply:
no, this patch goes to trunk which will be 5.1. This will never happen to be in any 5.0.x release.

Reply:
let's put this in 5.0.x

IIUC there's no compatibility break here.  it's an internal operational improvement, that's permitted in patch releases.  

is it possible to keep the 5.0 yaml unchanged, and introduce the new in trunk ?   this would be in spirit of no visible user-facing change.  we have existing compatibility in place for this type of string→parameterised class change.

IMO let's keep this PR focused on the 5.0.x patch.

Reply:
@tiagomlalves or we can also do it for 5.0.x, right? This is just fine IMHO. Let me check it once more.